### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by roopeshvs** [vinayak-mehta/present#52]
```bash
>present sample.md
Traceback (most recent call last):
  File "c:\python38\lib\runpy.py", line 194, in _run_module_as_main      
    return _run_code(code, main_globals, None,
  File "c:\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python38\Scripts\present.exe\__main__.py", line 4, in <module>
  File "c:\python38\lib\site-packages\present\cli.py", line 6, in <module>
    from .slideshow import Slideshow
  File "c:\python38\lib\site-packages\present\slideshow.py", line 8, in <module>
    from asciimatics.screen import Screen, _CursesScreen
ImportError: cannot import name '_CursesScreen' from 'asciimatics.screen' (c:\python38\lib\site-packages\asciimatics\screen.py)
```

_CursesScreen is only UNIX-compatible.
asciimatics also provides the use of _WindowsScreen class can be used instead for Windows-based systems.

Python's native curses doesn't directly support Windows.
We can use this port instead https://pypi.org/project/windows-curses/ for Windows-based systems.

This allows present to run on Windows!

**Comment by vinayak-mehta** [comment]
Thanks for reporting this, I'll push a quick fix to remove italics support for now, until it can be done in a cross-platform way in the future.

**Comment by vinayak-mehta** [comment]
This should be fixed with present v0.5.1, please reopen it if you still face the issue.
